src/app/CommandSender.cpp

```diff
@@ -291,6 +303,16 @@ CHIP_ERROR CommandSender::ProcessInvokeResponse(System::PacketBufferHandle && pa
         ReturnErrorOnFailure(ProcessInvokeResponseIB(invokeResponse));
     }
 
+    // No need to check error from GetMoreChunkedMessages if the call fails. `moreChunkedMessages` is already set
+    // to the default to use when MoreChunkedMessages is not provided in InvokeResponseMessage.
```

Comment:
That assumes that on failure, it's impossible internal consistency checks were missed so that `moreChunkedMessages` != false on failure.